recording_external_revenue/models/recording_external_revenue_raw.py

```diff
@@ -425,3 +425,12 @@ def _map_product_from_revenue_type(self):
     def _map_tax(self):
         if self.tax:
             return self.env["recording.tax.mapping"].map(self.company_id, self.tax)
+
+    def _map_analytic_account(self, recording):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-recording-addons/pullRequest?prid=5757477)